resources/public/scenarios/individual-income/HAPPY_PATH_2.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pensionsAnnuitiesAndOtherStateBenefits": {
+    "otherPensionsAndRetirementAnnuities": 36.5,
+    "incapacityBenefit": 980.45,
+    "jobseekersAllowance": 89.99
```

Comment:
`pensionsAnnuitiesAndOtherStateBenefits` should be an empty object as these three fields are optional. Please remove these fields, thanks